### PR TITLE
Fix #141 & Fix #1392: Update topic description

### LINKED ISSUE #141: Final UI: Topic overview tab fragment [Blocked: #15, #136]
This is tracking implementing the high fidelity UI for the topic overview tab.

https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/19cfbacf-854c-4c7d-8691-3b3d117e1866/TP-Overview-

## Task List

- [x] Topic thumbnail (Image) with gradient over image (without topic-name) (Assigned to @veena14cs )
Screenshot 2019-10-31 at 6 17 29 PM

- [ ] Topic content including topic-name, story-count, topic description
Screenshot 2019-10-31 at 6 17 37 PM

- [x] See More text  and Download Status (Assigned to @veena14cs )
Screenshot 2019-11-07 at 3 30 29 PM
Screenshot 2019-11-07 at 3 29 24 PM

- [ ] Final integration and final touch-up

- [ ] Accessibility Scanner is working correctly (https://support.google.com/accessibility/android/answer/6376570?hl=en)

### LINKED ISSUE #1392: Remove rich-text support for Topic Description
Currently topic description in `TopicInfo` screen supports rich-text, this needs to a simple text and for doing that following steps needs to be done:
1. Update existing description to remove richtext and change it to plain text.
2. Update lowfi functionality to remove usage of `HtmlParser`.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #141 & #1392 

Currently in develop, the topic description in InfoFragment supports rich text and this PR removes that support because topic description will be plain text only.

Note: I have just removed one particular test case in this and if you want to run/check the existing test cases you can run that specifically on Nexus S device because at the time of development of this PR that device was finalised for testing purposes. On other high-end devices these test cases will fail and it can only be fixed once #1379 is finished.

## Before
Screenshot_1592992943

## After
Screenshot_1592991802

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by veena14cs — APPROVED** [review]
LGTM.

**Comment by anandwana001** [comment]
@rt4914 What if the text is a bit long like this? 
is this good to go or we should have some max length limit and leave some space at the end to look it good?
Screenshot_1593004123

File: app/src/main/java/org/oppia/app/topic/info/TopicInfoFragmentPresenter.kt [hunk 1]
```diff
@@ -82,12 +82,7 @@ class TopicInfoFragmentPresenter @Inject constructor(
       fragment,
       Observer<Topic> { topic ->
         topicInfoViewModel.topic.set(topic)
-        topicInfoViewModel.topicDescription.set(
-          htmlParser.parseOppiaHtml(
-            topic.description,
-            fragment.requireView().findViewById(R.id.topic_description_text_view)
-          )
-        )
+        topicInfoViewModel.topicDescription.set(topic.description)
```
Comment (anandwana001) [inline app/src/main/java/org/oppia/app/topic/info/TopicInfoFragmentPresenter.kt:85]: This looks good. 
Only line number 123 has a ktlint issue. `private fun logInfoFragmentEvent(topicId: String){` single space before `{`

File: app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt [hunk 2]
```diff
@@ -86,22 +83,6 @@ class TopicInfoFragmentTest {
     }
   }
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt:85]: Same here, missing space ktlint issue. Line number 60. 
`fun setUp(){`

**Review by anandwana001 — APPROVED** [review]
LGTM.
nit changes to prevent `ktlint` failure.

File: app/src/main/java/org/oppia/app/topic/info/TopicInfoFragmentPresenter.kt (hunk 1 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/topic/info/TopicInfoFragmentPresenter.kt:85]: Done.

File: app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt (hunk 2 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt:85]: Done.

**Comment by rt4914** [comment]
> @rt4914 What if the text is a bit long like this?
> is this good to go or we should have some max length limit and leave some space at the end to look it good?
> Screenshot_1593004123

This is correct, text length has been controlled correctly in app as well as the oppia backend.

**Pull request merged**